### PR TITLE
preparation for verification by email or phone

### PR DESCRIPTION
**Pull request by QuyDang1108**
preparation for verification by email or phone
add to a new field in user entity which holds code verification
modify several variables in constants and use it

### CODE REVIEW

File: src/main/java/com/fjb/sunrise/controllers/AuthController.java [hunk 1]
```diff
@@ -1,10 +1,15 @@
 package com.fjb.sunrise.controllers;
 
 import static com.fjb.sunrise.utils.Constants.ApiConstant.AUTH_REDIRECT_LOGIN;
+import static com.fjb.sunrise.utils.Constants.ApiConstant.AUTH_VERIFICATION;
```
Comment (khanhduzz) [inline src/main/java/com/fjb/sunrise/controllers/AuthController.java:4]: Consider removing these constants from the import area and use 'Constants.ApiConstant...' within the code body instead.

File: src/main/java/com/fjb/sunrise/controllers/AuthController.java [hunk 2]
```diff
@@ -25,42 +30,76 @@ public class AuthController {
...
-            modelAndView.addObject(ERROR_MESSAGE, "Đăng kí không thành công");
+            modelAndView.addObject(ERROR_MESSAGE_OBJECT, "Đăng kí không thành công");
         }
 
         return modelAndView;
     }
 
+    //this function is prepared in advance and will be rewritten later.
+    @PostMapping("/sendToEmail")
+    public ModelAndView doSendCodeToEmail(@ModelAttribute(EMAIL_OBJECT) String email) {
```
Comment (khanhduzz) [inline src/main/java/com/fjb/sunrise/controllers/AuthController.java:90]: Are there any dependencies required for sending an email or a phone message?

**Comment by khanhduzz** [comment]
If you complete your tasks soon, consider writing unit tests for the service and controller to increase the overall code coverage.

File: src/main/java/com/fjb/sunrise/controllers/AuthController.java (hunk 2 above)
Comment (QuyDang1108) [reply src/main/java/com/fjb/sunrise/controllers/AuthController.java:90]: I will add it immediately

**Review by khanhduzz — APPROVED** [review]

**Pull request closed without merging**